runtime/tools/python/ttrt/common/run.py

```diff
@@ -430,6 +442,24 @@ def _execute(binaries):
                                 total_inputs.append(inputs)
                                 total_outputs.append(outputs)
 
+                                self.logging.debug(
+                                    f"generating golden map for loop={loop+1}/{self['--loops']} for binary={bin.file_path}"
+                                )
+                                goldens = {}
+
+                                for key, golden_obj in program.golden_map.map.items():
+                                    goldens[key] = ttrt.runtime.create_tensor(
+                                        golden_obj.torch_tensor.data_ptr(),
+                                        list(golden_obj.tensor_shape),
+                                        list(golden_obj.tensor_stride),
+                                        golden_obj.torch_tensor.element_size(),  # 4 bytes - float32
+                                        Binary.Program.to_data_type(
+                                            golden_obj.torch_tensor.dtype
+                                        ),
+                                    )
+
+                                total_goldens.append(goldens)
```

Comment:
What is the purpose of `total_goldens`, `goldens` and the above routine, it doesn't appear to be used?

Reply:
legacy from old implementation. removed